src/pdf/ShoppingList/ShoppingListButton.tsx

```diff
@@ -0,0 +1,24 @@
+"use client";
+
+import React from "react";
+import { ShoppingListPDFProps } from "@/pdf/ShoppingList/dataPreparation";
+import { NoSsr } from "@mui/material";
+import { PDFDownloadLink } from "@react-pdf/renderer";
+import ShoppingListPDF from "@/pdf/ShoppingList/ShoppingListPDF";
+
+interface Props {
+    data: ShoppingListPDFProps;
+    text: string;
+}
+
+const ShoppingListButton: React.FC<Props> = ({ data, text }) => {
+    return (
+        <NoSsr>
+            <PDFDownloadLink document={<ShoppingListPDF {...data} />} fileName="ShoppingList.pdf">
```

Comment:
I think Varun has mentioned this to my pdf branch - it might be nice to not have a fixed ShoppingList name but maybe link it to some parcel details to make list of different parcel have different names

Reply:
yep - I think @paopaosoftwire is implementing that when he does a generic pdf thing